### ISSUE TITLE
Does the densification trigger actually include view-space gradients, or just the edge-aware score?

### ISSUE THREAD

**Issue by deivse** [XiaoBin2001/Improved-GS#9]
In this excerpt from the current version of the paper, it states that you "use the view-averaged absolute coordinate gradients as the second criterion for evaluation." But I can't really find that anywhere in the code. Would  you be able to clarify if the results in the paper are using some combination of both, or just the edge-aware score? 

<img width="485" height="418" alt="Image" src="https://github.com/user-attachments/assets/399f5b3e-2d6e-446a-aa38-a58c0d858d34" />

Additionally, I wanted to ask about the hyperparameter changes reported in appendix:

> Regarding hyperparameters, we adjusted position lr init to 0.00004 and position lr final to 0.000002. The gradient threshold
> used in EAS is set to 0.0003 and the opacity reset threshold is set 0.05.

Have you evaluated your method without these changes? How does it affect the results? Is there any motivation for the position LR changes?

**Comment by deivse** [comment]
Sorry for the initial confusion, I've now noticed that only the gaussians that have both a high edge-aware score and position gradients are densified, although I would still appreciate some clarification regarding the learning rate changes,

**Comment by XiaoBin2001** [comment]
Regarding the variation in learning rate, it can be simply understood that our densification method requires less post-processing optimization adjustment, especially for positional adjustments, as it is well-designed, allowing for the use of a smaller learning rate. In practice, using the original learning rate yields similar results.
